src/main/kotlin/no/nav/familie/ef/sak/selvstendig/NæringsinntektKontrollService.kt

```diff
@@ -0,0 +1,132 @@
+package no.nav.familie.ef.sak.selvstendig
+
+import no.nav.familie.ef.sak.amelding.InntektService
+import no.nav.familie.ef.sak.behandling.BehandlingService
+import no.nav.familie.ef.sak.behandling.domain.Behandling
+import no.nav.familie.ef.sak.fagsak.FagsakService
+import no.nav.familie.ef.sak.felles.util.kuttPeriodeTilGittÅr
+import no.nav.familie.ef.sak.oppgave.OppgaveService
+import no.nav.familie.ef.sak.oppgave.OppgaveUtil
+import no.nav.familie.ef.sak.sigrun.SigrunService
+import no.nav.familie.ef.sak.vedtak.VedtakService
+import no.nav.familie.kontrakter.felles.Behandlingstema
+import no.nav.familie.kontrakter.felles.Månedsperiode
+import no.nav.familie.kontrakter.felles.Tema
+import no.nav.familie.kontrakter.felles.ef.StønadType
+import no.nav.familie.kontrakter.felles.oppgave.FinnOppgaveRequest
+import no.nav.familie.kontrakter.felles.oppgave.IdentGruppe
+import no.nav.familie.kontrakter.felles.oppgave.Oppgave
+import no.nav.familie.kontrakter.felles.oppgave.Oppgavetype
+import no.nav.familie.leader.LeaderClient
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+import java.time.LocalDate
+import java.time.YearMonth
+import java.util.UUID
+
+@Service
+class NæringsinntektKontrollService(
+    val oppgaveService: OppgaveService,
+    val fagsakService: FagsakService,
+    val behandlingService: BehandlingService,
+    val vedtakService: VedtakService,
+    val sigrunService: SigrunService,
+    val inntektService: InntektService,
+) {
+    private val secureLogger = LoggerFactory.getLogger("secureLogger")
+    private val årstallIFjor = YearMonth.now().minusYears(1).year
```

Comment:
Mulig jeg er kjip, kjipesen her - men denne kan bli misvisende etter nyttår og inntil vi har deployet ny versjon av appen :D

Reply:
Hehe, men den gjør jo ingenting etter at appen er initialisert. Dessuten blir dette en engangsjobb. Har bare satt opp at den skal kjøre etter at app er initialisert for testing og lite spam i logger sin del.